src/packages/flavors/index.ts

```diff
@@ -52,9 +52,18 @@ export function GetConnector(
         return new connector(providerOptions, executor);
       } catch (e) {
         if (e.message.includes("Cannot find module '@ganache/filecoin'")) {
-          throw new Error(
-            "Could not find module @ganache/filecoin peer dependency; please run `npm install @ganache/filecoin`"
+          // we print and exit rather than throw to prevent webpack output from being
+          // spat out for the line number
+          const bold = "\x1b[1m";
+          const red = "\x1b[31m";
+          const blue = "\x1b[34m";
+          const reset = "\x1b[0m";
+          console.warn(
+            `\n\n${bold}${red}ERROR:${reset} Could not find module ${bold}@ganache/filecoin${reset} peer dependency; ` +
+              `please run "${bold}${blue}npm install @ganache/filecoin${reset}" if you're using as a library ` +
+              `or "${bold}${blue}npm install --global @ganache/filecoin${reset}" if you're using in the Ganache CLI.\n\n`
```

Comment:
Some wonderful bikeshedding ahead! :grin: 

What do you think of changing this to:


```javascript
const { TruffleColors } = require("./src/packages/colors");
const chalk = require("chalk");

const NEED_HELP = "Need help? Reach out to the Truffle community at";
const COMMUNITY_LINK = "https://gitter.im/ConsenSys/truffle";

const bold = "\x1b[1m";
const red = "\x1b[31m";
const blue = "\x1b[34m";
const reset = "\x1b[0m";

console.warn(
  `\n\n${bold}${red}ERROR:${reset} Could not find Ganache flavor "${bold}filecoin${reset}" (@ganache/filecoin); ` +
    `it probably\nneeds to be installed.\n` +
    ` ▸ if you're using Ganache as a library run: \n` +
    `   ${bold}${blue}$ npm install @ganache/filecoin${reset}\n` +
    ` ▸ if you're using Ganache as a CLI run: \n` +
    `   ${bold}${blue}$ npm install --global @ganache/filecoin${reset}\n\n` +
    chalk`{hex("${TruffleColors.porsche}").bold ${NEED_HELP}}\n` +
    chalk`{hex("${TruffleColors.turquoise}") ${COMMUNITY_LINK}}\n\n`
);

```

renders as:

![image](https://user-images.githubusercontent.com/187813/111348863-ffb08500-8656-11eb-84c8-b5b4a10f60e5.png)

Reply:
I love every part of it except of the Gitter link :P (as a continual, rhetorical complaint that we should have discontinued that Gitter channel a long time ago)

Looks **much** better. Thanks for putting the time into formatting it to something much cleaner. No bikeshedding necessary!